### DISCUSSION TITLE
Are the DevTools available for all projects or limited in use?

### DISCUSSION THREAD

**Discussion by hagealex** [Laravel-Backpack/community-forum#425] [Q&A (Help)] [answered]
I'm thinking about purchasing the DevTools addon. But I'm nor sure if the usage is limited to one project or if I can use it for all my projects?

For example:
I have one project that uses a PRO license and one without. Could I use the DevTools in both projects?

**Comment by pxpm** [comment, marked as answer]
Hey @Alexander-Hagemeier 

There is **no limit** on the usage of Devtools or projects where it can be installed.
That said, you can generate any models, controllers etc to any amount of projects you are building with a single license of Devtools, just use the same credentials on every project. 

I never get tired to mention, Devtools is a developer dependency, and should **never be used in production** environments. 🔴 

Let me know if you have other questions. 

Cheers

**Reply by hagealex** [reply]
@pxpm Thank you for the clarification! :)
